doc/connectivity/region_extraction.rst

```diff
@@ -0,0 +1,189 @@
+
+Fetching resting state functional datasets
+==========================================
+
+We use ADHD resting state functional connectivity datasets of 20 subjects,
+which is already preprocessed and publicly available at
+`<http://fcon_1000.projects.nitrc.org/indi/adhd200/>`_. We use utilities
+:func:`fetch_adhd` implemented in nilearn for automatic fetching of these
+datasets.
+
+
+.. literalinclude:: ../../examples/connectivity/plot_extract_regions_canica_maps.py
+    :start-after: # utilities
+    :end-before: ################################################################################
+
+.. currentmodule:: nilearn.decomposition
+
+Data decomposition using Canonical ICA
+======================================
+
+Here, we use :class:`CanICA`, a multi subject model to decompose previously
+fetched multi subjects datasets. We do this by setting the parameters in the
+object and calling fit on the functional filenames without necessarily
+converting each filename to Nifti1Image object.
+
+
+.. literalinclude:: ../../examples/connectivity/plot_extract_regions_canica_maps.py
+    :start-after: # decomposition module
+    :end-before: # Visualization
+
+.. currentmodule:: nilearn.plotting
+
+Visualization of Canonical ICA maps
+===================================
+
+Showing ICA maps stored in components_img using nilearn plotting utilities.
+Here, we use :func:`plot_prob_atlas` for easy visualization of 4D atlas maps
+onto the anatomical standard template. Each ICA map is displayed in different
+color and colors are random and automatically picked.
+
+.. literalinclude:: ../../examples/connectivity/plot_extract_regions_canica_maps.py
+    :start-after: # Show ICA maps by using plotting utilities
+    :end-before: ################################################################################
+
+.. image:: ../auto_examples/connectivity/images/sphx_glr_plot_extract_regions_canica_maps_001.png
+    :target: ../auto_examples/connectivity/plot_extract_regions_canica_maps.html
+    :scale: 60
+
+.. currentmodule:: nilearn.regions
+
+Region Extraction with CanICA maps
+==================================
+
+We use object :class:`RegionExtractor` for extracting brain connected regions
+from ICA maps into separated brain activation regions with automatic
+thresholding strategy selected as thresholding_strategy='ratio_n_voxels'. We use
+thresholding strategy to first get foreground information present in the maps and
+then followed by robust region extraction on foreground information using
+Random Walker algorithm selected as extractor='local_regions'. The minimum
+number of voxels we keep is by defining min_region_size=1350 in mm^3 which means
+input should be given according to the volume of the voxel. Please see the
+documentation of nilearn.regions.connected_regions for more details.
+
+More explanation about threshold, threshold=0.5 can keep the nominal amount of
+non-zero voxels in brain volume across all the maps. If you need to keep more
```

Comment:
threshold=.5 represents the expected proportion of voxels included in the regions (i.e. with a non-zero value in one  of the maps).